tests/tracking/test_tracking.py

```diff
@@ -547,14 +546,12 @@ def test_log_artifact_with_dirs(tmpdir):
         ) == {os.path.basename(str(sub_dir))}
 
 
-def test_log_artifact():
-    artifact_src_dir = tempfile.mkdtemp()
+def test_log_artifact(tmp_path):
```

Comment:
Do we just need to revert the changes in this test and manually clean up the temp directory? Looks like a sqlite instance got created which causes the diff to not match?

Reply:
@BenWilson2 

Then, we can create a subdirectory in `tmp_path`, thanks for the catch!